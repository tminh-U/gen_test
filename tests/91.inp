8
13 10 1 4 8 8 11 2 
12
3 3
5 5
4 8
2 4
5 8
5 8
8 8
4 7
2 4
1 7
1 3
8 8
