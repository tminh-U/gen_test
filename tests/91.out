2
2
17
5
9
9
2
16
5
35
4
2
